### PR TITLE
zulip_bots: Check API key on initialization for Link Shortener bot.

### PR DESCRIPTION
**Pull request by ViRu-ThE-ViRuS**
Issue #171

### CODE REVIEW

**Comment by ViRu-ThE-ViRuS** [comment]
I had to override the `test_bot_responds_to_empty_message` in order to not fail any unit tests.
The bot does not exit when wrong API key is used. It just logs the message. If I make it exit, the bot starts failing its unit tests.

File: zulip_bots/zulip_bots/bots/link_shortener/test_link_shortener.py [hunk 1]
```diff
@@ -7,6 +7,9 @@ def _test(self, message: str, response: str) -> None:
         with self.mock_config_info({'key': 'qwertyuiop'}):
             self.verify_reply(message, response)
 
+    def test_bot_responds_to_empty_message(self) -> None:
+        self._test('', 'No links found. Send "help" to see usage instructions.')
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/link_shortener/test_link_shortener.py]: This test should go into its own commit.

File: zulip_bots/zulip_bots/bots/link_shortener/link_shortener.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import re
 import requests
+import logging
+import sys
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/link_shortener/link_shortener.py]: The sys import doesn't seem to be needed for this commit.

**Comment by ViRu-ThE-ViRuS** [comment]
@roberthoenig Thanks for your response. Fixing the issues. :-)

**Comment by ViRu-ThE-ViRuS** [comment]
@roberthoenig Done!

**Comment by roberthoenig** [comment]
Merged, after adding some detail to the second commit message. Thanks @ViRu-ThE-ViRuS! You won't see a "Merged" label on this branch because I merged your commits by checking them out in my own branch so I could modify the commit message. You can look in the commit history of python-zulip-api and find your commits there :)
